.stylelintrc.json

```diff
@@ -1,37 +1,37 @@
 {
-  "extends": "stylelint-config-standard-scss",
-  "plugins": [
-    "stylelint-use-logical-spec"
-  ],
-  "rules": {
-	"at-rule-empty-line-before": null,
-	"declaration-empty-line-before": null,
-	"custom-property-empty-line-before": null,
-	"comment-empty-line-before": null,
-	"scss/double-slash-comment-empty-line-before": null,
-	"scss/dollar-variable-colon-space-after": null,
+    "extends": "stylelint-config-standard-scss",
+    "plugins": [
+        "stylelint-use-logical-spec"
+    ],
+    "rules": {
+        "at-rule-empty-line-before": null,
+        "declaration-empty-line-before": null,
+        "custom-property-empty-line-before": null,
+        "comment-empty-line-before": null,
+        "scss/double-slash-comment-empty-line-before": null,
+        "scss/dollar-variable-colon-space-after": null,
 
-	"custom-property-pattern": null,
-	"declaration-block-no-duplicate-properties": null,
-	"declaration-block-no-redundant-longhand-properties": null,
-	"declaration-block-no-shorthand-property-overrides": null,
-	"declaration-block-single-line-max-declarations": null,
-	"font-family-no-duplicate-names": null,
-	"function-url-quotes": null,
-	"keyframes-name-pattern": null,
-	"no-descending-specificity": null,
-	"no-duplicate-selectors": null,
-	"scss/at-extend-no-missing-placeholder": null,
-	"scss/at-import-partial-extension": null,
-	"scss/at-mixin-pattern": null,
-	"scss/comment-no-empty": null,
-	"scss/dollar-variable-pattern": null,
-	"scss/double-slash-comment-whitespace-inside": null,
-	"scss/no-global-function-names": null,
-	"scss/operator-no-unspaced": null,
-	"selector-class-pattern": null,
-	"selector-id-pattern": null,
+        "custom-property-pattern": null,
+        "declaration-block-no-duplicate-properties": null,
+        "declaration-block-no-redundant-longhand-properties": null,
+        "declaration-block-no-shorthand-property-overrides": null,
+        "declaration-block-single-line-max-declarations": null,
+        "font-family-no-duplicate-names": null,
+        "function-url-quotes": null,
+        "keyframes-name-pattern": null,
+        "no-descending-specificity": null,
+        "no-duplicate-selectors": null,
+        "scss/at-extend-no-missing-placeholder": null,
+        "scss/at-import-partial-extension": null,
+        "scss/at-mixin-pattern": null,
+        "scss/comment-no-empty": null,
+        "scss/dollar-variable-pattern": null,
+        "scss/double-slash-comment-whitespace-inside": null,
+        "scss/no-global-function-names": null,
+        "scss/operator-no-unspaced": null,
+        "selector-class-pattern": null,
+        "selector-id-pattern": null,
 
-	"liberty/use-logical-spec": "always"
-  }
+        "liberty/use-logical-spec": "always"
+    }
```

Comment:
I preferred the mess! Especially as every cockpit plugin does things different, different rules.